packages/upward-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@magento/upward-js",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
@zetlen @dpatil-magento why did `upward-js` and `upward-spec` stay as `2.2.0` rather than be bumped to `3.0.0` with the rest of the packages?

Reply:
Those two packages didn't have any breaking changes, so they only got a minor bump. Our packages are individually versioned and published, this is just the first time they had different bumps.